src/EPR.SubsidiaryBulkUpload.Application/Models/BulkUpdateErrors.cs

```diff
@@ -39,4 +39,16 @@ public static class BulkUpdateErrors
 
     public const int InvalidDatafoundinRow = 114;
     public const string InvalidDatafoundinRowMessage = "Extra column value in the file.";
+
+    public const int ResourceNotFoundError = 118;
+    public const string ResourceNotFoundErrorMessage = "Information cannot be retrieved. Try again later.";
+
+    public const int ResourceNotReachableError = 119;
+    public const string ResourceNotReachableErrorMessage = "There is a problem with our service. Try again later.";
+
+    public const int InvalidFieldLengthError = 120;
+    public const string InvalidFieldLengthErrorMessage = "Field length is invalid. 8 Characters allowed.";
```

Comment:
This would be better named InvalidCompaniesHouseLengthErrorMessage

Reply:
changed to InvalidCompaniesHouseNumberLengthErrorMessage